--- Pootle-2.0.4/local_apps/pootle_app/views/admin/adminpages.py~	2009-11-24 15:19:43.000000000 +0200
+++ Pootle-2.0.4/local_apps/pootle_app/views/admin/adminpages.py	2010-06-09 21:36:27.825828157 +0300
@@ -72,7 +72,7 @@
 
     if not depcheck.test_iso_codes():
         optional.append({'dependency': 'iso-codes',
-                           'text': _("Can't find the ISO codes package. Pootle uses ISO codes to translate language names.")})
+                           'text': _("Can't find languages from ISO codes package. Be sure that languages configured are actually installed in iso-codes package. Pootle uses ISO codes to translate language names.")})
 
     if not depcheck.test_lxml():
         optional.append({'dependency': 'lxml',
