--- Pootle-2.7.3/pootle/checks.py	2016-04-25 21:29:00.000000000 +0300
+++ Pootle-2.7.3.rfc2606/pootle/checks.py	2016-05-06 23:06:34.627360978 +0300
@@ -196,7 +196,7 @@
             id="pootle.W008",
         ))
 
-    if settings.POOTLE_CONTACT_EMAIL in ("info@YOUR_DOMAIN.com") \
+    if settings.POOTLE_CONTACT_EMAIL in ("info@example.com") \
        and settings.POOTLE_CONTACT_ENABLED:
         errors.append(checks.Warning(
             _("POOTLE_CONTACT_EMAIL is using the following default "
@@ -214,7 +214,7 @@
             id="pootle.W009",
         ))
 
-    if settings.DEFAULT_FROM_EMAIL in ("info@YOUR_DOMAIN.com",
+    if settings.DEFAULT_FROM_EMAIL in ("info@example.com",
                                        "webmaster@localhost"):
         errors.append(checks.Warning(
             _("DEFAULT_FROM_EMAIL is using the following default "
--- Pootle-2.7.3/pootle/settings/30-site.conf	2016-04-25 17:04:49.000000000 +0300
+++ Pootle-2.7.3.rfc2606/pootle/settings/30-site.conf	2016-05-06 23:06:40.610978106 +0300
@@ -22,7 +22,7 @@
 ALLOWED_HOSTS = ['127.0.0.1', 'localhost']
 
 # Address to receive messages sent by contact form.
-POOTLE_CONTACT_EMAIL = 'info@YOUR_DOMAIN.com'
+POOTLE_CONTACT_EMAIL = 'info@example.com'
 POOTLE_CONTACT_ENABLED = True
 
 # By default Pootle uses SMTP server on localhost, if the server is
--- Pootle-2.7.3/pootle/settings/90-local.conf.template	2016-04-25 21:29:00.000000000 +0300
+++ Pootle-2.7.3.rfc2606/pootle/settings/90-local.conf.template	2016-05-06 23:07:12.862507728 +0300
@@ -139,24 +139,24 @@
 
 # This Pootle server admins
 ADMINS = (
-    # ('Your Name', 'your_email@domain.com'),
+    # ('Your Name', 'your_email@example.com'),
 )
 
 # Mail settings
 
 # Default email address to use for messages sent by Pootle.
 # https://docs.djangoproject.com/en/dev/ref/settings/#default-from-email
-DEFAULT_FROM_EMAIL = 'info@YOUR_DOMAIN.com'
+DEFAULT_FROM_EMAIL = 'info@example.com'
 
 # Address to receive messages sent by contact form.
-POOTLE_CONTACT_EMAIL = 'info@YOUR_DOMAIN.com'
+POOTLE_CONTACT_EMAIL = 'info@example.com'
 POOTLE_CONTACT_ENABLED = True
 
 # Email address to report string errors to, unless a report email was set for
 # the project for which the string error is being reported.
 # Use this as a way to forward string error reports to an address
 # different from the general contact address.
-POOTLE_CONTACT_REPORT_EMAIL = 'string_errors_manager@YOUR_DOMAIN.com'
+POOTLE_CONTACT_REPORT_EMAIL = 'string_errors_manager@example.com'
 
 # Mail server settings
 
@@ -165,7 +165,7 @@
 # external outgoing SMTP server.
 
 # Example for Google as an external SMTP server
-#EMAIL_HOST_USER = 'USER@YOUR_DOMAIN.com'
+#EMAIL_HOST_USER = 'USER@example.com'
 #EMAIL_HOST_PASSWORD = 'YOUR_PASSWORD'
 #EMAIL_HOST = 'smtp.gmail.com'
 #EMAIL_PORT = 587
--- Pootle-2.7.3/pootle/settings/60-translation.conf~	2016-04-25 17:04:49.000000000 +0300
+++ Pootle-2.7.3/pootle/settings/60-translation.conf	2016-05-06 23:42:45.773331232 +0300
@@ -5,7 +5,7 @@
 
 
 # The directory where the translation files are kept
-POOTLE_TRANSLATION_DIRECTORY = working_path('translations')
+POOTLE_TRANSLATION_DIRECTORY = '/var/lib/pootle/po'
 
 # On POSIX systems, files synchronized to disk will be assigned these
 # permissions. Use 0644/0755 for publically-readable files, or 0600/0700 if you
--- Pootle-2.7.3/pootle/settings/25-logging.conf~	2016-04-25 21:29:00.000000000 +0300
+++ Pootle-2.7.3/pootle/settings/25-logging.conf	2016-05-08 11:06:50.064007062 +0300
@@ -3,7 +3,7 @@
 #
 
 # The directory where Pootle writes its logs
-POOTLE_LOG_DIRECTORY = working_path("log")
+POOTLE_LOG_DIRECTORY = '/var/log/pootle'
 
 # Useful references:
 #
--- Pootle-2.7.3/pootle/settings/20-backends.conf~	2016-04-25 17:04:49.000000000 +0300
+++ Pootle-2.7.3/pootle/settings/20-backends.conf	2016-05-08 11:13:33.265043075 +0300
@@ -8,7 +8,7 @@
 DATABASES = {
     'default': {
         'ENGINE': 'transaction_hooks.backends.sqlite3',
-        'NAME': working_path('dbs/pootle.db'),
+        'NAME': '/var/lib/pootle/dbs/pootle.db',
         'USER': '',
         'PASSWORD': '',
         'HOST': '',
