--- Pootle-2.1.5/localsettings.py~	2010-12-29 08:59:26.000000000 +0200
+++ Pootle-2.1.5/localsettings.py	2011-01-31 12:01:13.958278122 +0200
@@ -82,7 +82,7 @@
 #SESSION_ENGINE = 'django.contrib.sessions.backends.cache'
 
 # Uncomment this if you're using memcached as CACHE_BACKEND and running under Django >= 1.1
-#SESSION_ENGINE = 'django.contrib.sessions.backends.cached_db'
+SESSION_ENGINE = 'django.contrib.sessions.backends.cached_db'
 
 # To improve performance, non-logged users get cached copies of most pages.
 # This variable is the number of seconds for which a page will be reused from
